metpy/plots/tests/test_declarative.py

```diff
@@ -108,8 +108,10 @@ def test_declarative_events():
     # Update some properties to make sure it regenerates the figure
     contour.linewidth = 2
     contour.linecolor = 'green'
+    contour.linetype = 'dashed'
```

Comment:
I'm confused why with these changes this test isn't failing?

Reply:
Fixed the tests and added more. I wasn't running with the --mpl option, so I wasn't actually testing the figures!